dm/master/config.go

```diff
@@ -228,3 +288,59 @@ func (c *Config) Reload() error {
 
 	return c.adjust()
 }
+
+// genEmbedEtcdConfig generates the configuration needed by embed etcd.
+func (c *Config) genEmbedEtcdConfig() (*embed.Config, error) {
+	cfg := embed.NewConfig()
+	cfg.Name = c.Name
+	cfg.Dir = c.DataDir
+
+	// reuse the previous master-addr as the client listening URL.
+	cURL, err := parseURLs(c.MasterAddr)
+	if err != nil {
+		return nil, terror.ErrMasterGenEmbedEtcdConfigFail.Delegate(err, "--listen-client-urls/--advertise-client-urls")
```

Comment:
we don't have `--listen-client-urls` or `--advertise-client-urls` in dm's config, the user may don't know how to solve this error, how about making it clear to let the user know the `MasterAddr` is wrong

Reply:
updated in https://github.com/pingcap/dm/pull/344/commits/31436e8155c1fac9c59e8a2bc5d78c8399d651a6.